gulpfile.js

```diff
@@ -2,7 +2,7 @@ var gulp = require('gulp');
 var eslint = require('gulp-eslint');
 
 var raptorSrc = [
-  './*.js',
+  './index.js',
```

Comment:
Rather it is being moved into the lib directory so it can be run along with everything else.